admin/class-automation.php

```diff
@@ -170,68 +209,14 @@ public static function get_automation_rules(): array {
 	}
 
 	/**
-	 * A render callback for the automation settings field.
-	 */
-	public static function render_settings_field(): void {
-		$value = self::get_automation_rules();
-		// TODO: Dynamically get list of registered taxonomies.
-		?>
-			<fieldset>
-				<legend class="screen-reader-text">
-					<?php esc_html_e( 'Configuration', 'apple-news' ); ?> 1
-				</legend>
-				<div>
-					<label>
-						<?php esc_html_e( 'Taxonomy', 'apple-news' ); ?>
-						<select name="<?php echo esc_attr( self::OPTION_KEY ); ?>[0][taxonomy]">
-							<option value=""></option>
-							<option value="category" <?php selected( 'category' === ( $value[0]['taxonomy'] ?? '' ) ); ?>><?php esc_html_e( 'Category', 'apple-news' ); ?></option>
-							<option value="post_tag" <?php selected( 'post_tag' === ( $value[0]['taxonomy'] ?? '' ) ); ?>><?php esc_html_e( 'Tag', 'apple-news' ); ?></option>
-						</select>
-					</label>
-				</div>
-				<div>
-					<label>
-						<?php esc_html_e( 'Term ID', 'apple-news' ); ?>
-						<input name="<?php echo esc_attr( self::OPTION_KEY ); ?>[0][term_id]" type="number" value="<?php echo esc_attr( $value[0]['term_id'] ?? '' ); ?>" />
-					</label>
-				</div>
-				<div>
-					<label>
-						<?php esc_html_e( 'Field', 'apple-news' ); ?>
-						<select name="<?php echo esc_attr( self::OPTION_KEY ); ?>[0][field]">
-							<option value=""></option>
-							<?php foreach ( array_keys( self::FIELDS ) as $field_key ) : ?>
-								<option value="<?php echo esc_attr( $field_key ); ?>" <?php selected( ( $value[0]['field'] ?? '' ) === $field_key ); ?>><?php echo esc_html( $field_key ); ?></option>
-							<?php endforeach; ?>
-						</select>
-					</label>
-				</div>
-				<div>
-					<label>
-						<?php esc_html_e( 'Field Value', 'apple-news' ); ?>
-						<input name="<?php echo esc_attr( self::OPTION_KEY ); ?>[0][value]" type="text" value="<?php echo esc_attr( $value[0]['value'] ?? '' ); ?>" />
-					</label>
-				</div>
-			</fieldset>
-		<?php
-	}
-
-	/**
-	 * A render callback for the submenu page.
+	 * A render callback for the React submenu page.
 	 */
 	public static function render_submenu_page(): void {
-		?>
-			<div class="wrap apple-news-settings">
-				<h1><?php echo esc_html( get_admin_page_title() ); ?></h1>
-				<?php settings_errors(); ?>
-				<form method="post" action="options.php" id="apple-news-automation">
-					<?php settings_fields( 'apple-news-automation' ); ?>
-					<?php do_settings_sections( 'apple-news-automation' ); ?>
-					<?php submit_button(); ?>
-				</form>
-			</div>
-		<?php
+		// Calling this here instead of in init hook to limited to this submenu
+		self::action__admin_print_scripts();
```

Comment:
Recommend just moving the code from `action__admin_print_scripts` here. We aren't actually using it as an action, and it's only applicable here, so might as well co-locate.